resources/qml/ActionButton.qml

```diff
@@ -11,7 +11,8 @@ import UM 1.1 as UM
 Button
 {
     id: button
-    property alias iconSource: buttonIcon.source
+    property alias iconSource: buttonIconLeft.source
+    property var iconOnRightSide: false
```

Comment:
```suggestion
    property bool iconOnRightSide: false
```
Also: I think the name should be a bit different (isIconOnRightSide) or (IconSide & use an enum)

Reply:
Done here: 2fdfdaa00b4cc7cea04e754a25ad58f056d3d684 (and also changed the data type).